src/js/services/forms.js

```diff
@@ -89,6 +105,9 @@ export default App.extend({
     return Promise.resolve(Radio.request('entities', 'fetch:directories:model', directoryName, query))
       .then(directory => {
         channel.request('send', 'fetch:directory', { value: directory.get('value'), requestId });
+      })
+      .catch(error => {
```

Comment:
@zfixler also I added a error to `getDirectory` to match the others so your `try/catch`es on `getDirectory` should reflect the API error